api/interservice/notifications/service/events.proto

```diff
@@ -0,0 +1,173 @@
+syntax = "proto3";
+
+package chef.automate.domain.notifications.service;
+
+option go_package = "github.com/chef/automate/api/interservice/notifications/service";
+
+////
+// Notifications messages and responses
+////
+
+
+message ExceptionInfo {
+  string class = 1;
+  string title = 2;
+  string msg = 3;
+  repeated string backtrace = 4;
+}
+
+message TimeInfo {
+  string start_time = 1;
+  string end_time = 2;
+}
+
+message Profile {
+  message Control {
+
+    message Result {
+      string status = 1;
+      string code_desc = 2;
+      float run_time = 3;
+      string start_time = 4;
+      string message = 5;
+      string skip_message = 6;
+    }
+
+    message ResultTotals {
+      int32 num_tests = 1;
+      int32 num_failed_tests = 2;
+      int32 num_skipped_tests = 3;
+      int32 num_passed_tests = 4;
+    }
+
+    string id = 1;
+    float impact = 2;
+    string title = 3;
+    string code = 4;
+    string desc = 5;
+    SourceLocation source_location = 6;
+    repeated Refs refs = 7;
+    reserved 8;// google.protobuf.Struct tags = 8;
```

Comment:
Is this for future use, or leftover, or what? Not sure what the commented item is supposed to be conveying here.

Reply:
This is a straight copy of what was previously in the notifications-service's source tree. I have opted to make fewer changes here for now in order to ensure that the generated go code for the proto package will be a drop-in replacement, so that the cutover from the old notifications service to the new one will be a simple as possible. I would like to do a cleanup pass on these protos after that.